src/content/Overlay/overlay.playground.jsx

```diff
@@ -13,7 +13,7 @@ const OverlayPlayground = () => {
     props: {
       children: {
         type: PropTypes.ReactNode,
-        value: `<Text>Some content</Text><TouchableOpacity onPress={()=>setIsVisible(!isVisible)}><Text>Click to close</Text></TouchableOpacity>`,
```

Comment:
removing touchable opacity will cause a problem when fullscreen is checked. there would be no option to go out.